news-bundle/src/Resources/contao/modules/ModuleNewsReader.php

```diff
@@ -125,7 +125,15 @@ protected function compile()
 			case 'external':
 				if ($objArticle->url)
 				{
-					throw new RedirectResponseException($objArticle->url, 301);
+					$url = System::getContainer()->get('contao.insert_tag.parser')->replaceInline($objArticle->url);
+
+					// Make the URL absolute
+					if (!preg_match('@^https?://@i', $url))
+					{
+						$url = Environment::get('base') . ltrim($url, '/');
+					}
```

Comment:
```suggestion
					$url = UrlUtil::makeAbsolute($url, Environment::get('base'));
```

Reply:
Changed in 7ffccdf143c54a59fab808dedd690975b9baff30.